src/lib/utils.ts

```diff
@@ -33,3 +33,24 @@ export async function extractJiraTicket (data: string): Promise<string> {
     }
     return '';
 }
+
+export async function extractNote (data: string): Promise<string> {
+    const regEx = await getJiraTicketRegEx();
+    if (regEx) {
+        return data.replace(regEx, '').replace(/^[\s:-]+|[\s:-]+$/g, '');
+    }
+    return data;
```

Comment:
Do we remove the JIRA ticket for length?

Reply:
Yes, we remove the Jira ticket since we have a separate field to store this data already.  Usually a Jira ticket is prepend to the title for readability.